game.py

```diff
@@ -0,0 +1,90 @@
+# Here is the description of the game simulation.
+
+# this is a two player card game
+# the game starts with a deck of cards
+# the cards are dealt out to both players
+# on each turn:
+# both players turn over their top-most card
+# the player with the higher valued card takes the cards and puts them in their scoring pile (scoring 1 point per card)
+# this continues until the players have no cards left
+# the player with the highest score wins
+# It’s considered a simulation because the players don’t have any choice, dont worry about input
+import random
+
+class Card:
```

Comment:
This should not be Card. It's more like CardDeck.
But you still need a Card class, which contains self.types = ['spade', 'heart', 'diamond', 'club'].